pageserver/src/tenant.rs

```diff
@@ -516,11 +519,19 @@ pub(crate) enum ShutdownError {
     AlreadyStopping,
 }
 
-struct DeletionGuard(OwnedMutexGuard<bool>);
+pub struct DeletionGuard(OwnedMutexGuard<DeleteTimelineFlow>);
```

Comment:
pub not needed anymore after moving into the delete.

Reply:
Done https://github.com/neondatabase/neon/pull/4619/commits/51691e03c02416977c4521b2343f009556e6d68e